README.md

```diff
@@ -7,114 +7,43 @@ system (CMS). It is a deployable solution that leverages the power of AWS servic
 
-## Table of Contents
+## Deploy VideoHeadline
 
-- [Getting started](#getting-started)
-  - [Global configuration (local and AWS environment)](#global-configuration-local-and-aws-environment)
-    - [Prerequisites](#prerequisites)
-    - [Create .env file](#create-.env-file)
-    - [AWS Configuration](#aws-configuration)
-  - [Running the application in local environment](#running-the-application-in-local-environment)
-    - [Set up and running the application](#set-up-and-running-the-application)
-    - [AWS Services Configuration in the Admin Web](#aws-services-configurationiin-the-admin-web)
-  - [Running the application in AWS environment](#running-the-application-in-aws-environment)
-  - [Production Environment Tasks (optional)](#production-environment-tasks)
-  - [Custom CSS Configuration for the Player in an Organization](#Custom-css-configuration-for-the-player-in-an-organization)
+Deploy a stable version of VideoHeadline using a pre-built docker-hub image for a quick and easy deployment process.
+Alternatively, you can build the image locally to deploy a customized version of VideoHeadline, changing the code and it's configuration files (see [Table of extra contents](#table-of-extra-contents) for more information).
 
-## Getting started
-### Global configuration (local and AWS environment)
-#### Prerequisites
+### Prerequisites
 
-- AWS Account: Necessary for hosting and delivering video content.
-- Docker and Docker compose: Video Headline runs inside Docker containers, so it is necessary to have Docker and Docker Compose installed.
-- Yarn and Node.js(v10): Required to deploy AWS configurations and build the playerReact component.
-- Python: Necessary for running Django and other Python-based tools.
-- AWS CLI: Useful for configuring and managing AWS services from the command line.
+- AWS account (it's not necessary to have any profile configured locally).
+- Docker running on your machine.
 
-#### Create .env file
+### Steps to deploy Videoheadline Infrastructure
 
-Create a .env file at the root of the project with all the variables defined in the .env-example file and their respective values.
+1. Build deployment image: `docker build -t video-headline-deploy .`
 
-#### AWS Configuration
-Video Headline requires some IAM roles and permissiones. To automate the configuration, there's CDK code to create a Stack with all the requirements.
+2. Run deployment: `docker run -e AWS_ACCESS_KEY_ID=... -e AWS_SECRET_ACCESS_KEY=... -e AWS_SESSION_TOKEN=... -e PROCESS=deploy -it video-headline-deploy`
 
-To deploy this stack, follow this steps:
-1. Navigate to the infrastructure directory.
-2. Run the command: `yarn cdk deploy AwsConfigurationStack`.
+   - AWS_ACCESS_KEY_ID: AWS access key identifier.
+   - AWS_SECRET_ACCESS_KEY: AWS secret access key.
+   - AWS_SESSION_TOKEN: AWS session token (if required).
 
-This deployment will set up:
+   These variables can be found in AWS Command line or programmatic access.
```

Comment:
Is this necessary?

Reply:
I think its necessary in case the user does not know where the variables are